build.gradle

```diff
@@ -263,7 +263,7 @@ pitest {
     historyOutputLocation = 'build/reports/pitest/fastermutationtestingoutput'
     outputFormats = ['XML', 'HTML']
     timestampedReports = false
-    mutationThreshold = 70
+    mutationThreshold = 65
```

Comment:
mutation test coverage must be 70% or above.

Reply:
this has been discussed and can now go ahead